template/macros.jinja

```diff
@@ -1,8 +1,7 @@
-{%- macro files_switch(prefix,
```

Comment:
Removed the need for `prefix`.

Reply:
@myii, I'm looking into this to incorporate at the systemd-formula,
but I use it here to distinguish multiple files:
systemd:networkd (has tofs)
systemd:timesyncd (has tofs) 
but both should have a different `prefix`
otherwise the files will be in the same place....